homeassistant/components/time_date/config_flow.py

```diff
@@ -0,0 +1,130 @@
+"""Adds config flow for Time & Date integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from datetime import timedelta
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import EntityPlatform
+from homeassistant.helpers.schema_config_entry_flow import (
+    SchemaCommonFlowHandler,
+    SchemaConfigFlowHandler,
+    SchemaFlowError,
+    SchemaFlowFormStep,
+)
+from homeassistant.helpers.selector import (
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+from homeassistant.setup import async_prepare_setup_platform
+
+from .const import CONF_DISPLAY_OPTIONS, DOMAIN, OPTION_TYPES
+from .sensor import TimeDateSensor
+
+_LOGGER = logging.getLogger(__name__)
+
+USER_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_DISPLAY_OPTIONS): SelectSelector(
+            SelectSelectorConfig(
+                options=[option for option in OPTION_TYPES if option != "beat"],
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key="display_options",
+            )
+        ),
+    }
+)
+
+
+async def validate_input(
+    handler: SchemaCommonFlowHandler, user_input: dict[str, Any]
+) -> dict[str, Any]:
+    """Validate rest setup."""
+    hass = handler.parent_handler.hass
+    if hass.config.time_zone is None:
+        raise SchemaFlowError("timezone_not_exist")
+    return user_input
+
+
+CONFIG_FLOW = {
+    "user": SchemaFlowFormStep(
+        schema=USER_SCHEMA,
+        preview=DOMAIN,
+        validate_user_input=validate_input,
+    )
+}
+
+
+class TimeDateConfigFlowHandler(SchemaConfigFlowHandler, domain=DOMAIN):
+    """Handle a config flow for Time & Date."""
+
+    config_flow = CONFIG_FLOW
+
+    def async_config_entry_title(self, options: Mapping[str, Any]) -> str:
+        """Return config entry title."""
+        return f"Time & Date {options[CONF_DISPLAY_OPTIONS]}"
```

Comment:
This title should be translated, but maybe that's not possible?

Reply:
Not possible to translate the config entry title as far as I know